pywin32_postinstall.py

```diff
@@ -702,13 +657,14 @@ def main():
 
     > python pywin32_postinstall.py -install
 
-    If you installed pywin32 via a .exe installer, this should be run
-    automatically after installation, but if it fails you can run it again.
+    This should be run automatically after installation,
+    but if it fails you can run it again.
 
-    If you installed pywin32 via PIP, you almost certainly need to run this to
+    Given EXE installers are no longer provided,
```

Comment:
```suggestion
    Given EXE installers are no longer provided,
    and wheel installs can't run postinstall scripts,
```
ref: https://github.com/pypa/packaging-problems/issues/64